lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -2178,7 +2180,7 @@ input[name="subject"] {
     }
   }
 
-  .white-listed-students {
+  .white-listed-students, .certificate-invalidation-history {
```

Comment:
can we make it simple as `invalidation-history` ? I think scss is already in `certificate`  section